public/components/ResultsDataTable.vue

```diff
@@ -449,6 +451,14 @@ export default Vue.extend({
         this.currentPage = 1;
       }
     },
+
+    pageItems(prev: TableRow[], curr: TableRow[]) {
```

Comment:
I am not sure if this function is needed for the table view. The table view uses visibleItems which when changed (from pagination or any other data transformation should trigger the refetch)